ElasticSearch/README.md

```diff
@@ -3,3 +3,48 @@
  # ElasticSearch xRunBooks
 
 * [Elasticsearch Rolling Restart](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master)
+
+
+ # ElasticSearch Actions 
+
+* Elasticsearch check health status.py
+* Elasticsearch cluster statistics.py
+* Elasticsearch cluster statistics.py
+* Elasticsearch delete unassigned shards.py
+* Elasticsearch disable shard allocation.py
+* Elasticsearch enable shard allocation.py
+* Elasticsearch get handle.py
+* Elasticsearch list allocations.py
+* Elasticsearch list nodes.py
+* Elasticsearch search query.py
+* Elasticsearch search query new.py
+
+
+ # ElasticSearch Actions 
+
+| ElasticSearch | [Elasticsearch check health status.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_check_health_status) 
+| ElasticSearch | [Elasticsearch cluster statistics.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_cluster_statistics) 
+| ElasticSearch | [Elasticsearch cluster statistics.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_cluster_statistics) 
+| ElasticSearch | [Elasticsearch delete unassigned shards.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_delete_unassigned_shards) 
+| ElasticSearch | [Elasticsearch disable shard allocation.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_disable_shard_allocation) 
+| ElasticSearch | [Elasticsearch enable shard allocation.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_enable_shard_allocation) 
+| ElasticSearch | [Elasticsearch get handle.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_get_handle) 
+| ElasticSearch | [Elasticsearch list allocations.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_list_allocations) 
+| ElasticSearch | [Elasticsearch list nodes.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_list_nodes) 
+| ElasticSearch | [Elasticsearch search query.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_search_query) 
+| ElasticSearch | [Elasticsearch search query new.py](https://github.com/unskript/Awesome-CloudOps-Automation/tree/master/ElasticSearch/legos/elasticsearch_search_query_new) 
+
```

Comment:
Why we have two entries?

Reply:
one is called _new.  Should we delete the "old" one?